nodewatcher/settings.py

```diff
@@ -726,4 +729,4 @@ def user_url(user):
 
 # Allowed hosts (required for production use, when DEBUG is false). Set it
 # to the (virtual HTTP) hostname under which you have nodewatcher installation.
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ['212.237.52.226']
```

Comment:
Please remove this as this is something related to your development environment.